### ISSUE TITLE
[MAYA-124459] Reference USD: useAsAnimationCache broken after namespace fix

### ISSUE THREAD

**Issue by jacobvfx** [Autodesk/maya-usd#2372]
**Describe the bug**
After the namespace bug was fixed #1307 the useAsAnimationCache flag on the usdImport translator no longer works.

**Steps to reproduce**

The following import code worked before the namespace fix:
```python
localPath = 'pointCache.usd'
namespace = 'pointCache'

cmds.file(localPath, namespace=namespace, mergeNamespacesOnClash=False,
		   reference=True, type='USD Import', ignoreVersion=True, sharedReferenceFile=False,
		   groupLocator=False, deferReference=False, lockReference=False,
	           options="assemblyRep=Import;readAnimData=1;useAsAnimationCache=1;")
```

**Expected behavior**
It should connect a pxrUsdPointBasedDeformer node to each shape in the reference but instead gives alot of Maya errors and does nothing.

**Attachments**
https://drive.google.com/file/d/1hW2a8YL6NPchLx0NPIWZLA1VDd1tJ-Ky/view?usp=sharing

**Specs (if applicable):**
 -  Linux
 -  Maya 2022.3
 -  Maya USD dev (21/05/2022)

**Comment by santosg87** [comment]
hi @jacobvfx. I have tried a few different things on my side, and, although i do see the issue you are mentioning, I seem to be able to reproduce it even before the changes from #1307 .

I have logged an issue internally to investigate this nonetheless.

**Comment by santosg87** [comment]
@jacobvfx i'm investigating this issue a bit deeper. small question - how were the original files created/your current pipeline? based on the nodes you are mentioning, i'm wondering if you guys are using the pxr scene assembly for your pipeline

**Comment by jacobvfx** [comment]
Hi Santos,

We are using both the PxrReferenceAssembly as regular Maya references of USD files. The issue is with the latter in this case.
The original file were just a simple export selected of a deforming mesh.

**Comment by jacobvfx** [comment]
any update on this one?

I can confirm that it works in 0.7.0, but not in 0.8.0.

**Comment by santosg87** [comment]
Hi @jacobvfx,

no big updates regarding this issue as of yet. we are investigating the case and have it schedule for future releases pf mayausd, although we have other issues that are higher in priority over this one.

thank you for the range between versions. this might help us narrow down a bit more on what could be the issue here.

**Comment by santosg87** [comment]
Hi @jacobvfx, a fix for this should be available with #2829. Marking this closed for the time being.

if you are able to compile mayaUSD with a newer commit that includes that PR, are you able to test it on your side?
if not, this fix should be available within the next release of mayaUSD.
